app/src/main/java/ingsw/group1/findmyphone/fragment/SettingsFragment.java

```diff
@@ -0,0 +1,34 @@
+package ingsw.group1.findmyphone.fragment;
+
+import android.os.Bundle;
+
+import androidx.preference.PreferenceFragmentCompat;
+
+import ingsw.group1.findmyphone.R;
+
+/**
+ * Fragment dedicated to showing the Preferences for the app.
+ *
+ * @author Riccardo De Zen
+ */
+public class SettingsFragment extends PreferenceFragmentCompat {
+
+    /**
+     * Default constructor.
+     */
+    public SettingsFragment() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5027057)